doc/org.eclipse.cdt.doc.user/src/getting_started/new_cbs_makefile_proj.adoc

```diff
@@ -0,0 +1,78 @@
+////
+Copyright (c) 2000, 2025 Contributors to the Eclipse Foundation
+This program and the accompanying materials
+are made available under the terms of the Eclipse Public License 2.0
+which accompanies this distribution, and is available at
+https://www.eclipse.org/legal/epl-2.0/
+
+SPDX-License-Identifier: EPL-2.0
+////
+
+// pull in shared headers, footers, etc
+:docinfo: shared
+
+// support image rendering and table of contents within GitHub
+ifdef::env-github[]
+:imagesdir: ../../images
+:toc:
+:toc-placement!:
+endif::[]
+
+// enable support for button, menu and keyboard macros
+:experimental:
+
+// Until ENDOFHEADER the content must match adoc-headers.txt for consistency,
+// this is checked by the build in do_generate_asciidoc.sh, which also ensures
+// that the checked in html is up to date.
+// do_generate_asciidoc.sh can also be used to apply this header to all the
+// adoc files.
+// ENDOFHEADER
+
+== Creating a Core Build System Makefile project
+
+This tutorial describes the process of creating a new C/{cpp} project that includes a Makefile file.
+
+You need to create a project to contain your source code and related files.
+A project has an associated builder that can incrementally compile source files as they are changed.
+
+To create a project:
+
+. Select menu:File[New > Project].
+
+
```

Comment:
I think multiple empty lines are being interpreted as the end of the numbered list. So the next numbered list item will be item "1" again. Try a single empty line only.

Reply:
I had some problems with keeping numbered lists together too, I ended up having to add `+` at the beginning of the blank lines to show list continuation. See https://docs.asciidoctor.org/asciidoc/latest/lists/continuation/#list-continuation